tests/ttnn/unit_tests/operations/test_moreh_layer_norm.py

```diff
@@ -440,14 +472,27 @@ def run_moreh_layer_norm_backward_with_gamma_or_beta(
         ([5, 2, 3, 4, 2 * TILE_HEIGHT + 13, 3 * TILE_WIDTH + 13], 4),  # test 6d
     ],
 )
-def test_moreh_layer_norm(input_shape_normalized_dims, elementwise_affine, eps, device):
+def test_moreh_layer_norm(input_shape_normalized_dims, elementwise_affine, eps, dtype, device):
     torch.manual_seed(2023)
-    run_moreh_layer_norm(input_shape_normalized_dims, elementwise_affine, eps, device)
+    if dtype == ttnn.bfloat8_b:
+        pytest.skip(f"bfloat8_b is not supported in the kernel")
```

Comment:
Why do you skip bfp8?
Is the test fails when using bfp8?

Reply:
The kernels don't support bfp8_b yet, so I'm skipping these tests for now